modules/xmpp/XmppConnection.js

```diff
@@ -409,6 +409,12 @@ export default class XmppConnection extends Listenable {
 
         return fetch(url)
             .then(response => {
+
+                // skips header checking if there is no info inn options
```

Comment:
typo `in`

Reply:
Now you reveal who is using the MacBook with the defective keyboard ...